sysfs/system_cpu.go

```diff
@@ -292,19 +296,93 @@ func parseCpufreqCpuinfo(cpuPath string) (*SystemCPUCpufreqStats, error) {
 		}
 	}
 
+	// "total_trans" is the total number of times the CPU has changed frequency.
+	var cpuinfoFrequencyTransitionsTotal *uint64 = nil
+	cpuinfoFrequencyTransitionsTotalUint, err := util.ReadUintFromFile(filepath.Join(cpuPath, "stats", "total_trans"))
+	if err != nil {
+		if !(os.IsNotExist(err) || os.IsPermission(err)) {
+			return &SystemCPUCpufreqStats{}, err
+		}
```

Comment:
Why are you not returning the error here but continue silently?

Reply:
https://github.com/prometheus/procfs/blob/9eed74490628b5d93360bd5e5971480c03fb43e0/sysfs/system_cpu.go#L273-L275

For these cases I saw that we were skipping over silently in other parts of the codebase, and kept the norm here.

Reply:
Ah I see, makes sense